--- /mnt/data/fw_pass02/CodeRedFactionWar_NoSC_MenuFree_Pass02/WorldResourceBridge/merge_preview/tune_root_tune_ai_game_main_original.tr
+++ /mnt/data/fw_pass02/CodeRedFactionWar_NoSC_MenuFree_Pass02/WorldResourceBridge/merge_preview/tune_root_tune_ai_game_main_with_codered.tr
@@ -5,10 +5,11 @@
 #include "audio.tr"
 
 
+#include "code_red_factionwar_world.tr"
 // Main human program
 program Main
 {
-	TRUE	->	Main0, PerformExternalGoals, UpdateHostility;
+	TRUE	->	Main0, PerformExternalGoals, UpdateHostility, CodeRedFactionWarWorld WITH 1.0;
 }
 
 
